src/annotator/test/annotation-sync-test.js

```diff
@@ -45,10 +37,10 @@ describe('AnnotationSync', () => {
 
   describe('#constructor', () => {
     context('when "deleteAnnotation" is published', () => {
```

Comment:
This is an issue with the existing code, but understanding these tests would be easier if the descriptions made it clear whether a particular event was an event coming from the sidebar or an event coming from the guest, like so:

```js
describe('handling events from sidebar', () => {
  describe('on "deleteAnnotation" event', ...)
});

describe('handling events from guest', () => {
  describe('on "beforeAnnotationCreated" event', ...);
});

The `describe('#constructor', ...)` block is not really useful. The constructor adds the listeners, but the tests are concerned with what happens when events are published rather than what directly happens when you call `new AnnotationSync`.